dev/tools/create_overlays.sh

```diff
@@ -44,6 +44,8 @@ OVERLAY_FILE=$(mktemp overlay-XXXX)
 # Create the overlay file
 > "$OVERLAY_FILE"
 
+bad=
```

Comment:
Maybe a slightly more explicit name would be in order, like "bad_repos" or something.

Reply:
Changed to `skipped_repos`